sdk/test-utils/perfstress/src/utils.ts

```diff
@@ -28,3 +30,40 @@ export async function drainStream(stream: NodeJS.ReadableStream) {
     stream.on("error", reject);
   });
 }
+export async function makeRequest(
+  uri: string,
+  requestOptions: RequestOptions
+): Promise<IncomingMessage> {
+  return new Promise<IncomingMessage>((resolve, reject) => {
+    const req = request(uri, requestOptions, resolve);
+
+    req.once("error", (err) => {
+      reject(new Error(err.message));
+    });
```

Comment:
why not simply reject with err?
```suggestion
    req.once("error", reject);
```

Reply:
:)